sdk/testing/src/main/java/com/mesosphere/sdk/testing/Expect.java

```diff
@@ -303,6 +295,35 @@ public String getDescription() {
         };
     }
 
+    /**
+     * Verifies that the indicated recovery phase.step has the expected status.
+     */
+    public static Expect recoveryStepStatus(String phaseName, String stepName, Status expectedStatus) {
```

Comment:
Could have a generic `stepStatus(String planName, String phaseName, ...)`, and then have `recoveryStepStatus(...)` just be a wrapper for that.

Also, could use `DefaultRecoveryPlanManager.DEFAULT_RECOVERY_PLAN_NAME`